src/librustc/query/mod.rs

```diff
@@ -663,7 +665,8 @@ rustc_queries! {
         /// associated types. This is almost always what you want,
         /// unless you are doing MIR optimizations, in which case you
         /// might want to use `reveal_all()` method to change modes.
-        query param_env(_: DefId) -> ty::ParamEnv<'tcx> {}
+        query param_env(_: DefId) -> ty::ParamEnv<'tcx> {
+        }
```

Comment:
Are these automatic formatting changes, or can they be removed?

Reply:
No, this is just an incomplete reversion. I've rebased these away.